WordPress/src/main/java/org/wordpress/android/ui/main/SitePickerActivity.java

```diff
@@ -256,8 +256,6 @@ public void onRefreshStarted() {
                             mSwipeToRefreshHelper.setRefreshing(false);
                             return;
                         }
-                        mDispatcher.dispatch(SiteActionBuilder.newRemoveWpcomAndJetpackSitesAction());
```

Comment:
With the removal of the sites Remove, the `onSiteRemoved()` callback seems like dead code, isn't it? I'd suggest deleting it and we can add it perhaps later if needed. Thanks!